### PR TITLE
feat: add TEST_BROWSER_DRIVER=playwright

### PR DESCRIPTION
**Pull request by hexsprite**

### CODE REVIEW

**Comment by jankapunkt** [comment]
I actually have no exprience with playwright. Are there any additional things to consider when using it? Specific installation steps?

File: browser/playwright.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+   } catch (error) {
+     console.error(error);
+     throw new Error(
+       'When running app tests with TEST_BROWSER_DRIVER=playwright, you must first ' +
+       '"npm i --save-dev playwright'
+     );
+   }
+ 
+   async function runTests() {
+     const browser = await playwright.chromium.launch();
```
Comment (Torgen) [inline browser/playwright.js:35]: Can there be an environment variable to choose between supported browsers instead of only allowing chromium?

Comment (jankapunkt) [reply browser/playwright.js:35]: @hexsprite I'n not into playwright yet. Can you answer on this?

Comment (hexsprite) [reply browser/playwright.js:35]: sounds like an additional feature... this is useful as is... if someone wants to add a PR for other browsers that could be a good addition but I don't think that would block this...

**Pull request merged**